src/SFML/Window/OSX/WindowImplCocoa.mm

```diff
@@ -436,6 +436,22 @@ void showMouseCursor()
 }
 
 
+////////////////////////////////////////////////////////////
+void WindowImplCocoa::setMinimumSize(const Vector2u& minimumSize)
+{
+    AutoreleasePool pool;
+    [m_delegate setMinimumSize:NSMakeSize(minimumSize.x, minimumSize.y)];
+}
+
+
+////////////////////////////////////////////////////////////
+void WindowImplCocoa::setMaximumSize(const Vector2u& maximumSize)
+{
+    AutoreleasePool pool;
+    [m_delegate setMaximumSize:NSMakeSize(maximumSize.x, maximumSize.y)];
```

Comment:
Does this consider the window style of no resize? Does MAC not allow windows to disable resizing?